backend/src/main/java/org/example/backend/Model/Unit.java

```diff
@@ -0,0 +1,10 @@
+package org.example.backend.Model;
+
+public enum Unit {
```

Comment:
I would leave the unit out for now, and keep a string as quantity, to keep things simple

Reply:
We can leave the units out if you prefer, but I think we should still make `Quantity` a number personally?